integrations/webchat/package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "watson-assistant-toolkit-for-web-chat",
+  "version": "1.0.0",
+  "description": "A set of examples for extending and customizing the web chat feature in Watson Assistant",
+  "scripts": {
+    "lint": "eslint \"./{examples,contributing}/**/*.{js,jsx}\"",
+    "lint:fix": "eslint \"./{examples,contributing}/**/*.{js,jsx}\" --fix",
+    "lint:fix:dryrun": "eslint \"./{examples,contributing}/**/*.{js,jsx}\" --fix-dry-run"
+  },
+  "devDependencies": {
+    "eslint": "^8.13.0",
+    "eslint-config-airbnb": "^19.0.4",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-jsdoc": "^39.2.1",
+    "eslint-plugin-jsx-a11y": "^6.5.1",
+    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-react": "^7.29.4",
+    "eslint-plugin-react-hooks": "^4.4.0",
+    "eslint-plugin-simple-import-sort": "^7.0.0",
+    "prettier": "^2.6.2"
+  },
+  "prettier": {
+    "singleQuote": true
+  },
+  "eslintConfig": {
```

Comment:
Hrmmm, would you mind setting this to eslint 7 instead of 8. I'm getting an error with IJ and google says that there's a bug with eslint 8 not working with the current version of IJ. And I wonder if it would be better putting this into an `.eslintrc` file.